d4s2_api/views.py

```diff
@@ -100,13 +102,25 @@ def send(self, request, pk=None):
         delivery = self.get_object()
         if not delivery.is_new():
             raise AlreadyNotifiedException(detail='Delivery already in progress')
-        accept_path = reverse('ownership-prompt') + "?token=" + str(delivery.token)
+        accept_path = reverse('ownership-prompt') + "?token=" + str(delivery.transfer_id)
```

Comment:
For consistency yes. I had left this as the abstract "token" rather than the concrete "transfer_id", since it's primarily there to make the URL unguessable. But that's not security,